dashboard/app/controllers/level_sources_controller.rb

```diff
@@ -1,3 +1,4 @@
+require 'base64'
```

Comment:
Done. Initially it got put here, as some of the model logic started in the controller.